ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -1122,8 +1208,35 @@ tflite_parseCustomOption (const GstTensorFilterProperties *prop, tflite_option_s
             option->delegate = TFLITE_DELEGATE_GPU;
           else if (g_ascii_strcasecmp (pair[1], "XNNPACK") == 0)
             option->delegate = TFLITE_DELEGATE_XNNPACK;
+          else if (g_ascii_strcasecmp (pair[1], "External") == 0)
+            option->delegate = TFLITE_DELEGATE_EXTERNAL;
           else
             ml_logw ("Unknown option to set tensorflow-lite delegate (%s).", pair[1]);
+        } else if (g_ascii_strcasecmp (pair[0], "ExtDelegateLib") == 0) {
+          option->ext_delegate_path = g_strdup (pair[1]);
+        } else if (g_ascii_strcasecmp (pair[0], "ExtDelegateKeyVal") == 0) {
+          gchar **kvpairs;
```

Comment:
In case the filter is reloaded, please initialize option->ext_delegate_path and option->ext_delegate_kv_table around LINE 1186.

Reply:
Explicit initialization added in updated patchset for clarity.

Though those variables were presumably already initialized from within tflite_loadModelFile() by statement:
`
tflite_option_s option = {};
`
prior to tflite_parseCustomOption() invokation.